Packs/PrismaCloud/Integrations/RedLock/RedLock.py

```diff
@@ -290,24 +290,16 @@ def get_alert_details():
         'ResourceAccessKeyAge': demisto.get(response, 'resource.additionalInfo.accessKeyAge'),
         'ResourceInactiveSinceTs': demisto.get(response, 'resource.additionalInfo.inactiveSinceTs')
     })
+    ec = alert_to_context(response)
+    ec['AlertRules'] = [alert_rule.get('name') for alert_rule in response.get('alertRules')]
```

Comment:
why isn't this part of the `alert_to_context` function?
```suggestion
    ec['AlertRules'] = [alert_rule.get('name') for alert_rule in response.get('alertRules', [])]
```